docs/fieldnorm/soups.md

```diff
@@ -0,0 +1,12 @@
+# SOUPS - Software of Unknown Provenance
+
+Software of unknown provenance (SOUPS) is a software item that is already developed, generally available and not developed for the purpose of being incorporated into the bioinformatic workflow, or a software item for which adequate software documentation is not available. A procedure should be documented about how and where SOUPS are registered.
+
+For each SOUP the following items should be documented:
+
+- Name: Name of SOUP.
+- Reference: Link (URL) to soup (source code / publication).
+- Intended purpose: Description of the original purpose of the SOUP.
+- Used according to the intended purpose: yes/no, if no, document for which purpose the SOUP is used.
+- Validation required: yes/no, often this is part of the validation of the (workflow) release itself. When using the SOUP outside of its intended purpose you need a proper validation to show that it works.
+- Version controlled: yes/no.
```

Comment:
What do you mean with version controlled exactly? normally you would incorporate a specific version of software and then not touch it again if it works, right? or do other people continuously update to newer versions of SOUPs? doesn't that require a lot of unnecessary validation work?